src/main/webapp/instructorHelp.jsp

```diff
@@ -27,13 +27,13 @@
               <a href="#faq4">How do I delete students from a course?</a>
             </li>
             <li>
-              <a href="#faq6">How do I change the Google ID associated with a student?</a>
+              <a href="#faq5">How do I change the Google ID associated with a student?</a>
             </li>
             <li>
-              <a href="#faq7">What to do if a student says his/her courses have disappeared from the system?</a>
+              <a href="#faq6">What to do if a student says his/her courses have disappeared from the system?</a>
             </li>
             <li>
-              <a href="#faq7a">How do I interpret/use contribution values in the results of ‘Team contribution’ type questions?</a>
+              <a href="#faq7">How do I interpret/use contribution values in the results of ‘Team contribution’ type questions?</a>
```

Comment:
Be careful when you change the name. I notice that you have change the `<li>`'s id accordingly. However, other html tag also refer to these id. Check these :)
![image](https://user-images.githubusercontent.com/13977235/34243189-98cd74a0-e65a-11e7-924b-7401e40c006a.png)
![image](https://user-images.githubusercontent.com/13977235/34243199-a1ef1d9a-e65a-11e7-86f7-142ef0336f33.png)